packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -133,7 +133,7 @@ export default class VTable {
             });
             if (this.formatInfo) {
                 saveTableInfo(this.table, this.formatInfo);
-                applyTableFormat(this.table, this.cells, this.formatInfo);
+                applyTableFormat(this.table, this.cells, this.formatInfo as Required<TableFormat>);

```

Comment:
you can change the type declaration in line 64 to be Required ..., so that no need to cast here. And also force us always assign a valid value to it.

Reply:
the line 111 
`this.formatInfo = getTableFormatInfo(this.table);`
is getting the formatInfo as TableFormat

Reply:
I know. And getTableFormatInfo() has validation code so it can guarantee the return value is a valid `Required<TableFormat>`. So you can change line 64 to enforce this and do not do type casting here.